.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -38,6 +38,10 @@ Please do not remove items from the checklist
 - [ ] Add a link to the tagged release in this issue.
 - [ ] Send an announcement email to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] node-feature-discovery $VERSION is released`
 - [ ] Add a link to the release announcement in this issue
+- [ ] For a major release, update README in master branch
+  - [ ] Update references to point to `$VERSION` and submit a PR
```

Comment:
what are these references ?

should this be : run `prepare_release.sh $VERSION <KEY ID>` and only commit the README ?

Reply:
> should this be : run `prepare_release.sh $VERSION <KEY ID>` and only commit the README ?
> 
> i assume it to replace any instance of current release with the new release e.g 0.7.0 -> 0.8.0

Basically yes, can be done with `prepare_release.sh`, too